packages/trees/frontend/sirius-components-trees/src/views/TreeView.types.ts

```diff
@@ -19,6 +19,7 @@ export interface TreeViewComponentProps extends WorkbenchViewComponentProps {
   showToolBar: boolean;
   enableMultiSelection: boolean;
   treeOptions: TreeOptions | undefined;
+  markedItemId?: string[] | undefined;
```

Comment:
Why optional?

Reply:
Some TreeView uses don't need to specify marked items, like in the explorer view.